Slambook/index.js

```diff
@@ -6,11 +6,26 @@ form = document.getElementsByTagName('form')[0];
 console.log(form,"form");
 
 // Add the evenlistner below
+document.querySelector('.form').addEventListener('submit', function(event) {
+    event.preventDefault(); // Prevent form submission

```

Comment:
Consider validating the user input before using it. Make sure to check if the input fields are not empty or if they contain valid data.